src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -75,7 +80,7 @@ class CreateFastaFromHvcf : CliktCommand(help = "Create a FASTA file from a h.vc
         rangeFasta
     }
 
-    val fastaType by option("--fasta-type", help = "Type of fasta exported.  Can be either composite or haplotype")
+    val fastaType by option("--fasta-type", help = "Type of fasta exported.")
```

Comment:
Should the help message be more specific to mention what types?  If given a list, the user could investigate further if they didn't understand what each type represented.